doc/md/predicates.md

```diff
@@ -89,6 +89,60 @@ client.Pet.
 
 Custom predicates can be useful if you want to write your own dialect-specific logic or to control the executed queries.
 
+For example, in order to use built-in SQL functions such as `DATE()`, use one of the following options:
+
+1. Pass a dialect-aware predicate function using the `sql.P` option:
+
+```go
+users := client.User.Query().
+	Select("id").
+	Where(P(func(b *Builder) {
+		b.WriteString("DATE(").Ident("last_login_at").WriteByte(')').WriteOp(OpGTE).Arg(value)
+	}))
```

Comment:
```suggestion
	Where(sql.P(func(b *sql.Builder) {
		b.WriteString("DATE(").Ident("last_login_at").WriteByte(')').WriteOp(OpGTE).Arg(value)
	})).
	AllX(ctx)
```

Reply:
Also, you can't use `sql.P` inside `UserQuery.Where`. I can edit this example later and land it if you prefer. Thanks